tmt/convert.py

```diff
@@ -46,6 +46,116 @@ def repr_str(dumper, data):
 #  Convert
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+def read_manual(plan_id, case_id, disabled):
+    """
+    Reads metadata of manual test cases from Nitrate
+    """
+    try:
+        import nitrate
+    except ImportError:
+        raise ConvertError('Install nitrate module to import metadata.')
+    # Turns off nitrate caching
+    nitrate.set_cache_level(0)
+
+    try:
+        tree = fmf.Tree(os.getcwd())
+    except fmf.utils.RootError:
+        raise ConvertError("Initialize metadata tree using 'tmt init'")
+
+    try:
+        if plan_id:
+            all_cases = nitrate.TestPlan(int(plan_id)).testcases
+            case_ids = [case.id for case in all_cases if case.manual]
+        else:
+            case_ids = [int(case_id)]
+    except ValueError:
+        raise ConvertError('Value of plan/case option must be an integer')
+
+    # Create directory to store manual tests in
+    old_cwd = os.getcwd()
+    os.chdir(tree.root)
+    try:
+        os.mkdir('Manual')
+    except FileExistsError:
+        pass
+
+    os.chdir('Manual')
+
+    for case_id in case_ids:
+        testcase = nitrate.TestCase(case_id)
+        if testcase.status.name != 'CONFIRMED' and not disabled:
+            continue
+
+        # Filename sanitization
+        dir_name = testcase.summary.replace(' ', '_')
+        dir_name = dir_name.replace('/', '_')
+        try:
+            os.mkdir(dir_name)
+        except FileExistsError:
+            pass
+
+        os.chdir(dir_name)
+        echo("Importing the '{0}' test case.".format(dir_name))
+
+        # Test case data
+        md_content = {}
+        md_content['Setup'] = testcase.setup
+        md_content['Actions'] = testcase.action
+        md_content['Breakdown'] = testcase.breakdown
+        md_content['Expected Results'] = testcase.effect
+
+        # Test case metadata
+        data = {}
+        data['manual'] = True
+        data['test'] = 'full.md'
+
+        # Relevancy
```

Comment:
It would be good to import just all nitrate attributes the same way as it is done in the `read` function. I'd suggest to create a new function for this shared stuff.

Reply:
Addressed in fb5ef79